### DISCUSSION TITLE
Terminei de estudar o basico sobre html - css e java script, e agora?

### DISCUSSION THREAD

**Discussion by RnStopped** [frontendbr/forum#2449] [Perguntas e Respostas]
ja vi varios videos, e desenvolvi algumas homes de sites, mas e agora? como fazer pra descobrir como fazer o resto funcionar?

**Comment by expalmer** [comment]
o que seria o `resto funcionar` ? tu diz o backend ?

**Comment by BertholiDan** [comment]
https://roadmap.sh/frontend

**Comment by wilsonneto-swe** [comment]
"Terminei de estudar X", isso não existe aqui hehe

Estudar é um processo contínuo que deve te acompanhar por toda a vida, inclusive HTML CSS, JS e etc, isso evolui smepre.

Mas bom, indo ao cerne da questao, o importante não é estudar, ou terminar um curso,  ou tutorial, mas sim entender e dominar aquilo, e a unica/melhor maneira para isso é, colocar em prática!

Faça alguns sites completos, mesmo que estaticos, isso vai ter o norte que precisa para os proximos passos.

O "e agora?" não seremos nós quem devemos te falar, mas sim os desafios que vc esbarrar praticando, trabalhando, no dia a dia... Por exemplo, vcoe fez um site estatico completo, mas precsia de um contato ali que envie o email, se nao sabe fazer, esta ai o seu "e agora?", entende?

Acho que é isso que deve te guiar, nao algum roadmap ou dicas nossas.
Faça algo, vai codando, desbravando e aprendendo o que precisar, é a melhor maneira e mais divertida.

No fim, não se apegue ou se limite a "estudar" ou terminar materiuais, se apegue a dominar aquilo, vcoe realmente domina HTML, CSS e JS?

Abracao, espero ajudar 
:)

Caso a resposta tenha ajudado, ou alguma outra dos amigos, lembre de marcar a resposta.

**Comment by Kauaguarinogabriell** [comment]
Cara, eu finalizei meus cursos de HTML, CSS e Javascript, e parti pro React, pretendo estudar Typescript, SASS, Vue e Bootstrap, é meio que um plano traçado ja

**Comment by nazarepiedady** [comment]
@RnStopped, estudar as bases não é o suficiente, porque precisas ser capaz de desenvolver usando apenas estas bases sem recorrer às abstrações, até fores capaz de o fazer, não estás pronto.
